aarch64_linux/build_aarch64_wheel.py

```diff
@@ -677,7 +681,7 @@ def parse_arguments():
     parser.add_argument("--build-only", action="store_true")
     parser.add_argument("--test-only", type=str)
     parser.add_argument("--os", type=str, choices=list(os_amis.keys()), default='ubuntu18_04')
-    parser.add_argument("--python-version", type=str, choices=['3.6', '3.7', '3.8', '3.9', '3.10', '3.11'], default=None)
+    parser.add_argument("--python-version", type=str, choices=['3.8', '3.9', '3.10', '3.11'], default=None)
```

Comment:
We don't really need to remove older versions (as the script is usable to build say PyTorch-1.10), but feel free to change the default